medical_ui/medical_ui/settings.py

```
@@ -0,0 +1,46 @@
+from pathlib import Path
+
+BASE_DIR = Path(__file__).resolve().parent.parent
+SECRET_KEY = "django-insecure-medical-ui-local-only"
```

Ruff (0.15.7)

[error] Possible hardcoded password assigned to: "SECRET_KEY"

(S105)